docs/protocol/ash-for-subnet-owners/overview.md

```diff
@@ -0,0 +1,35 @@
+---
+sidebar_label: Overview
+sidebar_position: 1
+---
+
+# Ash for Subnet Owners - Overview
+
+:::tip
+**Subnet owner:** An organization, whether public or private, that uses a dedicated Avalanche Subnet. The Subnet hosts one or more blockchains, serving as a backend for the business logic.
+:::
+
+Ash offers 2 validation services to Subnet owners:
+
+- Validation renting
+- Spontaneous validation by the protocol
+
+## Validation renting
+
+Subnet owners can **rent validation** from Ash nodes for a **fraction of the price** needed to bootstrap a validator.
+
+Instead of operating validator nodes themselves, Subnet owners **benefit from the expertise of the Ash team** (and Ash network of node operators in [V3](../roadmap)).
```

Comment:
```suggestion
Instead of operating validator nodes themselves, Subnet owners **benefit from the expertise of the Ash team** (and the Ash network of node operators in [V3](../roadmap)).
```